packages/ripple-keypairs/src/index.ts

```diff
@@ -88,6 +88,9 @@ function computePublicKeyHash(publicKeyBytes: Uint8Array): Uint8Array {
 }
```

Comment:
Is it possible to make a similar update to `computePublicKeyHash` method on line 86 ? That is another function making use of `publicKeyBytes`